packages/graphile-build-pg/src/plugins/PgBackwardRelationPlugin.js

```diff
@@ -64,7 +64,7 @@ export default (function PgBackwardRelationPlugin(
       return extend(
         fields,
         foreignKeyConstraints.reduce((memo, constraint) => {
-          if (omit(constraint, "read")) {
+          if (omit(constraint, "read") || omit(constraint, "many")) {
```

Comment:
This will make the `many` also apply to singular backward relations which I don't think is the correct behaviour. Can we move this check down to `shouldAddManyRelation`? If we do so then I think we can class this as a bugfix rather than a breaking change - since `@omit many` on the constraint didn't do anything before, but omitting the backward many relation would be the expected outcome.

Reply:
Hm, but I think we should have a way to omit singular backward relations as well (without omitting the forward relation), and `@omit many` is the best I can think of for that.
In #460 I proposed `@omit one` for the forward relation, but I don't know anything better than to use `@omit many` also on a one-to-one relation.

Reply:
No, if we kept this it'd be a breaking change because the result would be unexpected. Regarding `@omit one` I'd have to evaluate it. Lets put this on ice for now and come up with a full solution to #460 before we try and change what the smart comment omits mean too much.

Reply:
OK, I had changed it already to apply only on many-relationships, see the second commit. Until we come up with a solution, maybe emit a warning (error?) when there's an `@omit many` comment on a one-to-one constraint?